Command/DeleteUnusedFilesCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Svd\MediaBundle\Command;
+
+use Svd\CoreBundle\Command\BaseCommand;
+use Svd\MediaBundle\Entity\File;
+use Symfony\Bridge\Monolog\Logger;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Command
+ */
+class DeleteUnusedFilesCommand extends BaseCommand
+{
+    /**
+     * Configure
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('delete:unused:files');
+    }
+
+    /**
+     * Process
+     */
+    protected function process()
+    {
+        $fileRepository = $this->getContainer()
+            ->get('svd_media.file.repository');
+        $mediaManager = $this->getContainer()
+            ->get('svd_media.manager.media');
+
+        $files = $fileRepository->getBy([
```

Comment:
I think, we also need to add a separate method in fileRepository like:

public function iterateByNotUsed() {
        $qb = $this->createQueryBuilder('f')
            ->select('f');
        $result = $qb->getQuery()
            ->iterate();
}